website/package.json

```
@@ -69,8 +69,11 @@
     "@react-native-website/update-redirects": "0.0.0",
     "@types/google.analytics": "^0.0.46",
     "alex": "^10.0.0",
+    "case-police": "^0.5.14",
+    "eslint": "^8",
```

```suggestion
    "eslint": "^8.57.1",
```

Revert more granular range.